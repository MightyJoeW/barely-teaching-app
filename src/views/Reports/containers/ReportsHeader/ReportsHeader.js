import React from "react";
import { Link } from "react-router-dom";

const ReportsHeader = () => (
  <header className="reports-header">
    <div classsName="content-container">
      <Link className="reports-header__title" to="/reports">
        <h1> Student Behavior Reports </h1>
        <div className="reports-header__actions">
          <Link className="login-button" to="/reports/create">
            Create Report
          </Link>
        </div>
      </Link>
    </div>
  </header>
);

export default ReportsHeader;
