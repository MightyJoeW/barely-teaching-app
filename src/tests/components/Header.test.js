import React from "react";
import ReactShallowRenderer from "react-test-renderer/shallow";
import ReportsHeader from "../../views/Reports/containers/ReportsHeader/ReportsHeader";

test("should render Header correctly", () => {
  const renderer = new ReactShallowRenderer();
  renderer.render(<ReportsHeader />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
