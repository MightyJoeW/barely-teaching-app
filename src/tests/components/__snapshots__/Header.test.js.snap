// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div
  className="header"
>
  <div
    className="container"
  >
    <h1
      className="header__title"
    >
       Barely Teaching 
    </h1>
    <h2
      className="header__subtitle"
    >
      The #1 companion for jaded teachers
    </h2>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      to="/tasks"
    >
      Tasks
    </NavLink>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      to="/journal"
    >
      Journal
    </NavLink>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      to="/kidfree"
    >
      Kid-Free
    </NavLink>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      to="/reports"
    >
      Reports
    </NavLink>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      to="/help"
    >
      Help
    </NavLink>
    <button
      onClick={[Function]}
    >
       Logout 
    </button>
  </div>
</div>
`;
